docs/admin/templates/extending-templates/devcontainers.md

```diff
@@ -50,30 +80,94 @@ resource "coder_devcontainer" "my-repository" {
 
 ## Enable Dev Containers Integration
 
-To enable the dev containers integration in your workspace, you must set the
-`CODER_AGENT_DEVCONTAINERS_ENABLE` environment variable to `true` in your
-workspace container:
+Dev containers integration is **enabled by default** in Coder 2.24.0 and later.
+You don't need to set any environment variables unless you want to change the
+default behavior.
+
+If you need to explicitly disable dev containers, set the
+`CODER_AGENT_DEVCONTAINERS_ENABLE` environment variable to `false`:
 
 ```terraform
 resource "docker_container" "workspace" {
   count = data.coder_workspace.me.start_count
   image = "codercom/oss-dogfood:latest"
   env = [
-    "CODER_AGENT_DEVCONTAINERS_ENABLE=true",
+    "CODER_AGENT_DEVCONTAINERS_ENABLE=false",  # Explicitly disable
     # ... Other environment variables.
   ]
   # ... Other container configuration.
 }
 ```
 
-This environment variable is required for the Coder agent to detect and manage
-dev containers. Without it, the agent will not attempt to start or connect to
-dev containers even if the `coder_devcontainer` resource is defined.
+See the [Environment Variables](#environment-variables) section below for more
+details on available configuration options.
+
+## Environment Variables
+
+The following environment variables control dev container behavior in your
+workspace. Both `CODER_AGENT_DEVCONTAINERS_ENABLE` and
+`CODER_AGENT_DEVCONTAINERS_PROJECT_DISCOVERY_ENABLE` are **enabled by default**,
+so you typically don't need to set them unless you want to explicitly disable
+the feature.
+
+### CODER_AGENT_DEVCONTAINERS_ENABLE
+
+**Default: `true`** • **Added in: v2.24.0**
+
+Enables the dev containers integration in the Coder agent.
+
+```terraform
+env = [
+  "CODER_AGENT_DEVCONTAINERS_ENABLE=true",  # Enabled by default
+]
+```
+
+The dev containers feature is enabled by default. You can explicitly disable it
+by setting this to `false`.
+
+### CODER_AGENT_DEVCONTAINERS_PROJECT_DISCOVERY_ENABLE
+
+**Default: `true`** • **Added in: v2.25.0**
+
+Enables automatic discovery of dev containers in git repositories.
+
+```terraform
+env = [
+  "CODER_AGENT_DEVCONTAINERS_PROJECT_DISCOVERY_ENABLE=true",  # Enabled by default
+]
+```
+
+When enabled (the default), the agent will:
```

Comment:
This is kind of the third repeat of default (default: true, #enabled by default, the default). I think we only need to mention it once.